bundles/org.openhab.binding.mqtt.homeassistant/src/main/java/org/openhab/binding/mqtt/homeassistant/internal/component/Vacuum.java

```diff
@@ -69,23 +69,24 @@
             super("MQTT Vacuum");
         }
 
-        protected @Nullable String commandTopic;
-        protected String stateTopic = "";
-        protected @Nullable String sendCommandTopic; // for custom_command
+        // variable names are with "_" by intention to avoid code cluttering with serialization annotations
+        protected @Nullable String command_topic;
+        protected String state_topic = "";
```

Comment:
Did you see my comment? :)
```
// variable names are with "_" by intention to avoid code cluttering with serialization annotations
````
IMHO adding one annotation per variable is just spamming the code and makes it unreadable. If there would only be one or two such variables I would agree with you, but since there are more I would prefer to leave it like that.

Let me know what I should do, i.e. merge it or comment so I add some spam to the code... :)

Reply:
You can also call it future proofing instead of spam. 
It is not uncommon that the default (de)serialization strategies change over time. :wink:

Reply:
Fine. I have changed the MR to use annotations, please have a look again. Thanks.

Reply:
Thanks for changing it! The code might look less "spammy" if the DTO classes were in separate files. But that's a matter of personal taste.

Reply:
I agree, that was just copy&paste from other integrated homeassistant mqtt devices, hence all in one class+package ;)